src/VocabGamePlay.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class VocabGamePlay extends Component{
+    constructor(){
+		super();
```

Comment:
constructor and super should have props passed in
```
constructor(props) {
   super(props)
}
```

Reply:
done!